src/api/regex.c

```diff
@@ -101,17 +101,11 @@ int luaopen_regex(lua_State *L) {
   lua_setfield(L, -2, "__name");
   lua_pushvalue(L, -1);
   lua_setfield(L, LUA_REGISTRYINDEX, "regex");
-  lua_pushnumber(L, PCRE2_ANCHORED);
-  lua_setfield(L, -2, "ANCHORED");
-  lua_pushnumber(L, PCRE2_ANCHORED) ;
-  lua_setfield(L, -2, "ENDANCHORED");
-  lua_pushnumber(L, PCRE2_NOTBOL);
-  lua_setfield(L, -2, "NOTBOL");
-  lua_pushnumber(L, PCRE2_NOTEOL);
-  lua_setfield(L, -2, "NOTEOL");
-  lua_pushnumber(L, PCRE2_NOTEMPTY);
-  lua_setfield(L, -2, "NOTEMPTY");
-  lua_pushnumber(L, PCRE2_NOTEMPTY_ATSTART);
-  lua_setfield(L, -2, "NOTEMPTY_ATSTART");
+  API_CONSTANT_DEFINE(L, -1, "ANCHORED", PCRE2_ANCHORED);
```

Comment:
These changes here seems completely unrelated to the topic of this PR.

Reply:
Ah yes, sorry; this bled in from my other changes to process; I'll revert these.